cmd/agent/app/flare.go

```diff
@@ -45,6 +52,8 @@ var flareCmd = &cobra.Command{
 			color.NoColor = true
 		}
 
+		cpuProfURL = fmt.Sprintf("http://127.0.0.1:%s/debug/pprof/profile?seconds=%d", config.Datadog.GetString("expvar_port"), profiling)
```

Comment:
Would it be possible to get the `trace-agent` profiles too? The mechanism and paths are exactly identical, the only thing that would change is the port.

Reply:
Yeah would definitely be possible. Might have to leave it to a different PR though because I'm not sure I will get to adding that before friday, and I want to merge this before the freeze.